# Nils Freydank <holgersson@posteo.de> (2020-04-18)

Respect the gentoo paths for documentation - i.e. manual
installation of the documents.

--- a/setup.py  2020-04-18 19:19:30.000000000 +0200
+++ b/setup.py  2020-04-18 19:19:30.000000001 +0200
@@ -12,8 +12,7 @@
       packages=['urlscan'],
       scripts=['bin/urlscan'],
       package_data={'urlscan': ['assets/*']},
-      data_files=[('share/doc/urlscan', ['README.rst', 'COPYING']),
-                  ('share/man/man1', ['urlscan.1'])],
+      data_files=[('share/man/man1', ['urlscan.1'])],
       license="GPLv2",
       install_requires=["urwid>=1.2.1"]
       )

